cfme/tests/cli/test_appliance_update.py

```diff
@@ -88,3 +90,52 @@ def test_update_yum(appliance_preupdate, appliance):
     appliance_preupdate.evmserverd.start()
     appliance_preupdate.wait_for_web_ui()
     assert appliance.version == appliance_preupdate.version
+
+
+@pytest.fixture(scope="function")
+def enabled_embedded_appliance(appliance_preupdate):
+    """Takes a preconfigured appliance and enables the embedded ansible role"""
+    appliance_preupdate.enable_embedded_ansible_role()
+    assert appliance_preupdate.is_embedded_ansible_running
+    return appliance_preupdate
+
+
+@pytest.fixture(scope="function")
+def update_embedded_appliance(enabled_embedded_appliance, appliance):
+    with enabled_embedded_appliance:
+        red_hat_updates = RedHatUpdates(
+            service='rhsm',
+            url=conf.cfme_data['redhat_updates']['registration']['rhsm']['url'],
+            username=conf.credentials['rhsm']['username'],
+            password=conf.credentials['rhsm']['password'],
+            set_default_repository=True
+        )
+        red_hat_updates.update_registration(validate=False)
+        red_hat_updates.check_updates()
+        wait_for(
+            func=red_hat_updates.checked_updates,
+            func_args=[appliance.server.name],
+            delay=10,
+            num_sec=100,
+            fail_func=red_hat_updates.refresh
+        )
+        if red_hat_updates.platform_updates_available():
+            red_hat_updates.update_appliances()
+        return enabled_embedded_appliance
+
+
+@pytest.mark.uncollectif(lambda: not store.current_appliance.is_downstream)
```

Comment:
I approve the PR, but I'm not sure `fixtures.pytest_store.store`. @RonnyPfannschmidt please help

Reply:
The best solution would be to have marker like `downstream_only` for these purposes. @psav @quarckster @RonnyPfannschmidt ?

Reply:
@mfalesni we have `pytest.mark.ignore_stream("upstream"),`

Reply:
@lcouzens apply `pytest.mark.ignore_stream("upstream")` instead of `@pytest.mark.uncollectif(lambda: not store.current_appliance.is_downstream)` please.